🐞 - TS2552: Cannot find name 'PushPermissionState'. Did you mean 'PermissionState'?

### Playground Link

_No response_

### Description

After updating `@ng-web-apis/permissions` from 3.0.3 to 3.1.1 in my project, I'm getting the following errors when compiling the project:

```
Error: node_modules/@ng-web-apis/permissions/utils/permissions-predicates.d.ts:1:85 - error TS2552: Cannot find name 'PushPermissionState'. Did you mean 'PermissionState'?

1 export declare function isGranted(state: PermissionState | NotificationPermission | PushPermissionState): state is 'granted';
                                                                                      ~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ng-web-apis/permissions/utils/permissions-predicates.d.ts:2:84 - error TS2552: Cannot find name 'PushPermissionState'. Did you mean 'PermissionState'?

2 export declare function isDenied(state: PermissionState | NotificationPermission | PushPermissionState): state is 'denied';
                                                                                     ~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ng-web-apis/permissions/utils/permissions-predicates.d.ts:4:55 - error TS2552: Cannot find name 'PushPermissionState'. Did you mean 'PermissionState'?

4 export declare function isPrompt(s: PermissionState | PushPermissionState): s is 'prompt';
                                                        ~~~~~~~~~~~~~~~~~~~
```

I'm using TypeScript 5.1.6. `PushPermissionState` type has been removed from TypeScript in https://github.com/microsoft/TypeScript/commit/4e689bc075974bd1f70c00adc1df5634846e734e. 

### Angular version

16.2.7

### Taiga UI version

n/a

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [X] Linux
- [ ] iOS
- [ ] Android

--- a/libs/permissions/src/utils/permissions-predicates.ts
+++ b/libs/permissions/src/utils/permissions-predicates.ts
@@ -1,19 +1,19 @@
 export function isGranted(
-    state: PermissionState | NotificationPermission | PushPermissionState,
+    state: NotificationPermission | PermissionState,
 ): state is 'granted' {
     return state === 'granted';
 }
 
 export function isDenied(
-    state: PermissionState | NotificationPermission | PushPermissionState,
+    state: NotificationPermission | PermissionState,
 ): state is 'denied' {
     return state === 'denied';
 }
 
 export function isPrompt(s: NotificationPermission): s is 'default';
-export function isPrompt(s: PermissionState | PushPermissionState): s is 'prompt';
+export function isPrompt(s: PermissionState): s is 'prompt';
 export function isPrompt(
-    state: PermissionState | NotificationPermission | PushPermissionState,
+    state: PermissionState | NotificationPermission,
 ): state is 'prompt' | 'default' {
     return state === 'prompt' || state === 'default';
 }

